edm4hep.yaml

```
@@ -780,6 +780,36 @@ datatypes:
     OneToOneRelations:
       - edm4hep::Track track           // the corresponding track
 
+
+  #===== Generator related data =====
+
+  #---------- GeneratorEventParameters
+  edm4hep::GeneratorEventParameters:
+    Description: "Generator event parameters"
+    Author: "EDM4hep authors"
+    Members:
+      - double event_scale              // event scale
+      - double alphaQED                 // alpha_QED
+      - double alphaQCD                 // alpha_QCD
+      - int signal_process_id           // id of signal process
```

```suggestion
      - int signalProcessId           // id of signal process
```
Just to conform to our usual naming scheme